### PR TITLE
Talk about a TypeScript build error in the FAQ

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Comment by roj1512** [comment]
@KnorpelSenf Please give me feedback on the content, and tell me if it is in the right section with the right title.

File: site/docs/resources/faq.md [hunk 1]
```diff
@@ -115,6 +115,12 @@ There are a few things you can do.
 4. Come ask us in the group chat for help. We have experienced people there.
 5. It is possible to ask Telegram to increase the limits, but this is very unlikely to happen if you did not do steps 1-3 first.
 
+### Cannot find type definition file for 'node-fetch'.
+
+This is the result of a version conflict between the installed `@types/node-fetch`, and the grammY dependency `node-fetch`.
```
Comment (KnorpelSenf) [inline site/docs/resources/faq.md:120]:
```suggestion
This is the result of a version conflict between the installed `@types/node-fetch`, and the grammY dependency `node-fetch`.
```

Comment (KnorpelSenf) [inline site/docs/resources/faq.md:120]: It's actually just missing type information. There's no mismatch. We don't have the types at a production dependency, so they're missing if users try to type-check their node modules.

Skipping this step of checking everything is the best solution.

Alternatively, users can install the package `@types/node-fetch@2`.

Comment (roj1512) [reply site/docs/resources/faq.md]: I modified it.

File: site/docs/resources/faq.md [hunk 2]
```diff
@@ -115,6 +115,12 @@ There are a few things you can do.
 4. Come ask us in the group chat for help. We have experienced people there.
 5. It is possible to ask Telegram to increase the limits, but this is very unlikely to happen if you did not do steps 1-3 first.
 
+### Cannot find type definition file for 'node-fetch'.
+
+This is the result of some missing type declarations.
+
+The recommended way to fix this is to set `skipLibCheck` to `true` in your TypeScript compile options.
+
```
Comment (KnorpelSenf) [inline site/docs/resources/faq.md:123]:
```suggestion

If you are sure that you need this option to be kept to `false`, you can instead install the missing type definitions by running `npm i -D @types/node-fetch@2`.
```

**Comment by roj1512** [comment]
@KnorpelSenf for some reason the button to request another review is not working.

**Review by KnorpelSenf — APPROVED** [review]

File: site/docs/resources/faq.md [hunk 3]
```diff
@@ -115,6 +115,14 @@ There are a few things you can do.
 4. Come ask us in the group chat for help. We have experienced people there.
 5. It is possible to ask Telegram to increase the limits, but this is very unlikely to happen if you did not do steps 1-3 first.
 
+### Cannot find type definition file for 'node-fetch'.
```
Comment (WingLim) [inline site/docs/resources/faq.md:118]: According to https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md026

We should remove this punctuation.

**Review by WingLim — CHANGES_REQUESTED** [review]

File: site/docs/resources/faq.md (hunk 3 above)
Comment (KnorpelSenf) [reply site/docs/resources/faq.md:118]: I agree.
```suggestion
### Cannot find type definition file for 'node-fetch'
```

Comment (WingLim) [reply site/docs/resources/faq.md:118]: Or this is the orignal error message which we should keep it what it is?

Comment (roj1512) [reply site/docs/resources/faq.md:118]: It was the original message, but I think you can remove the dot. The meaning will stay there.

**Review by KnorpelSenf — APPROVED** [review]

**Review by WingLim — APPROVED** [review]

**Pull request merged**